regress/sql/expr.sql

```diff
@@ -675,6 +675,11 @@ $$) AS r(result agtype);
 SELECT * FROM cypher('expr', $$
 RETURN ([0, {one: 1, pie: 3.1415927, e: 2.718281::numeric}, 2::numeric, null])
 $$) AS r(result agtype);
+-- Testing float4 to agtype
+SELECT float4_to_agtype(42.42);
+SELECT float4_to_agtype(-42.42);
+SELECT float4_to_agtype(0.0);
+
```

Comment:
You should also add some tests for the cast itself, like:

SELECT 42.42::float4::agtype
ALSO be sure to check for NULL too.

Reply:
Done